DigitalLearningSolutions.Web.Tests/Helpers/LoginClaimsHelperTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Helpers
+{
+    using System.Security.Claims;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.Controllers;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class LoginHelperTests

```

Comment:
Nice tests, though could you just format them with Given-When-Then for ease of reading?